pages/faq.tsx

```diff
@@ -107,26 +107,30 @@ function FAQ({}: Props) {
         </FAQEntry>
         <FAQEntry title="I can't to open Ferdium after downloading. What should I do?">
           <>
-            This is because the releases aren&apos;t yet signed. We are still working on getting an Apple developer license and a certificate for Windows.
+            This is because the releases aren&apos;t yet signed. We are still working on getting an
+            Apple developer license and a certificate for Windows.
             <br />
             Meanwhile, you could check out these sections for{" "}
             <ExternalLink href='https://github.com/ferdium/ferdium-app#unsigned-nightlies-on-macos'>
-            MacOS
+              MacOS
             </ExternalLink>{" "}
-             and{" "}
+            and{" "}
             <ExternalLink href='https://github.com/ferdium/ferdium-app#unsigned-nightlies-on-windows'>
-            Windows
+              Windows
             </ExternalLink>{" "}
             in README.md to work around it.
           </>
         </FAQEntry>
-        <FAQEntry title="My Discord service is not showing properly. What should I do?">
+        <FAQEntry title='My Discord service is not showing properly. What should I do?'>
```

Comment:
Why is this change?

(reminder for later: setup `eslint` + `prettier` for these)

Reply:
Yeah, that's most likely prettier going at it :)

Reply:
I thought it was manual because https://github.com/ferdium/website/pull/19/files/647dc3db4f8cc05746eb575c781e07cd5105046c#diff-60fcdf7d2b47bb8b3bd7869a20d25049810c38525099d8a47e123f592c334bb5L108 used simple quotes, but I realise it's because this one as a legitimate simple quote inside.